metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -307,7 +324,7 @@ def tile_custom_properties(tile: PowerBiAPI.Tile) -> dict:
             lastModified=ChangeAuditStamps(),
             inputs=ds_input,
             externalUrl=tile.report.webUrl if tile.report else None,
-            customProperties={**tile_custom_properties(tile)},
+            customProperties=tile_custom_properties(tile),
```

Comment:
I wonder why we had copied it originally.

Reply:
by mistake, both statements does the same job, 327 is more readable.